packages/ripple-tide-api/src/services/tide-api-base.ts

```
@@ -52,16 +52,7 @@ export default class TideApiBase extends HttpClient {
       )
     }
 
-    try {
-      return await this.getMappedDataAux(mapping, resource)
-    } catch (error) {
-      throw new ApplicationError(
-        `An error occurred while mapping tide API data`,
-        {
-          cause: error
-        }
-      )
-    }
+    return await this.getMappedDataAux(mapping, resource)
```

Help me understand this - does removing the try catch return the mapping field error? Where is the error returned?

---

Catching and re-throwing it here was swallowing up the actual cause of the error when it is finally logged. There's more work we can do to improve logging but this fixes a common gripe, the intermediate error doesn't add much useful information.

Before:
```
ApplicationError: Error fetching site data
   ...stack trace
Caused by: ApplicationError: An error occurred while mapping tide API data
   ...unhelpful stack trace
```

After:
```
ApplicationError: Error fetching site data
   ...stack trace
Caused by: ReferenceError: footerSection is not defined
  ...helpful stack trace
```

---

The error instead gets caught at the api route level like all other errors: https://github.com/dpc-sdp/ripple-framework/blob/feature/R20-1507-social-links/packages/ripple-tide-api/src/utils/createHandler.ts#L59